server/socket/index.js

```diff
@@ -3,8 +3,12 @@ const socket = require('socket.io');
 let io;
 module.exports.listen = (http) => {
   io = socket.listen(http);
-  io.on('connection', () => {
+  io.on('connection', (thisSocket) => {
     console.log('Socket connected!');
+    thisSocket.on('text change', (newText) => {
+      console.log('sending new text:', newText);
+      thisSocket.broadcast.emit('serve text', newText);
+    });
```

Comment:
This looks good for now. These event listeners will be moved into a separate constructor function in the models in the near future.